src/App.tsx

```diff
@@ -1,16 +1,24 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [addMovie, setAddMovie] = useState(moviesFromServer);
+
+  const onAddNewMovie = (newMovie: Movie) => {
```

Comment:
```suggestion
  const handleAddNewMovie = (newMovie: Movie) => {
```

https://medium.com/@master_43681/understanding-and-proper-usage-of-on-and-handle-prefixes-in-react-function-naming-119f0beea400